weave-js/src/core/ops/domain/gql.ts

```diff
@@ -448,6 +448,8 @@ export const toGqlField = (
     return gqlBasicField('name');
   } else if (forwardOp.op.name === 'entity-isTeam') {
     return gqlBasicField('isTeam');
+  } else if (forwardGraph.getOp.name === 'entity-internalId') {
+    return gqlBasicField('id');
```

Comment:
This isn't used in this PR, but is a nice to have when we need to use it. I matched the `report-internalId` pattern here

Reply:
**question:** why are we calling it "internal" id?

Reply:
ids should already be fetched for each object and needn't be added to this section